hardware_interface/include/hardware_interface/resource_manager.hpp

```diff
@@ -69,8 +69,7 @@ class HARDWARE_INTERFACE_PUBLIC ResourceManager
    * activated. Currently used only in tests.
    */
   explicit ResourceManager(
-    const std::string & urdf, bool validate_interfaces = true, bool activate_all = false,
-    unsigned int update_rate = 100,
+    const std::string & urdf, bool activate_all = false, unsigned int update_rate = 100,
```

Comment:
This is missing a parameter removal as well. `validate_interfaces` has been removed, but is still documented. I cannot suggest a change since the line is outside of the diff.

Reply:
Agreed. This need to be fixed!

Reply:
What do you think now guys?